content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -283,14 +283,14 @@ volume.
 We can also take the snapshot using various options given by etcdctl. For example 
 
 ```shell
-ETCDCTL_API=3 etcdctl --h 
+ETCDCTL_API=3 etcdctl -h 
 ``` 
 
 will list various options available from etcdctl. For example, you can take a snapshot by specifying
 the endpoint, certificates etc as shown below:
 
 ```shell
-ETCDCTL_API=3 etcdctl --endpoints=[127.0.0.1:2379] \
+ETCDCTL_API=3 etcdctl --endpoints=https://[127.0.0.1]:2379 \
```

Comment:
Just a question: Is there any reason why we need to keep `[` and `]` here?

Reply:
Hi, I have verified, it works with both [ ] and also without them. Do you suggest me to remove it?

Reply:
I could not find the other place which contain `[` and `]` on this option.
If no problem, it is nice to remove it to avoid unnecessary confusion I feel.